go.mod

```diff
@@ -17,12 +17,12 @@ require (
 	github.com/go-logr/zapr v0.4.0
 	github.com/google/go-cmp v0.5.6
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
-	github.com/openshift/api v0.0.0-20210722202039-91e26cfb36fc
-	github.com/openshift/build-machinery-go v0.0.0-20210712174854-1bb7fd1518d3
-	github.com/openshift/library-go v0.0.0-20210331235027-66936e2fcc52
+	github.com/openshift/api v0.0.0-20210921141042-49b82c6d1ca9
+	github.com/openshift/build-machinery-go v0.0.0-20210806203541-4ea9b6da3a37
+	github.com/openshift/library-go v0.0.0-20210922102545-c1aa04539ce3
```

Comment:
Please make sure you only pick up the module changes you need.  I think this update to library-go is causing your PR to fail the unit-tests now:

```# github.com/openshift/cluster-ingress-operator/pkg/operator/controller/configurable-route
pkg/operator/controller/configurable-route/controller.go:339:42: not enough arguments in call to resourceapply.ApplyRole
	have ("k8s.io/client-go/kubernetes/typed/rbac/v1".RbacV1Interface, "github.com/openshift/library-go/pkg/operator/events".Recorder, *"k8s.io/api/rbac/v1".Role)
	want (context.Context, "k8s.io/client-go/kubernetes/typed/rbac/v1".RolesGetter, "github.com/openshift/library-go/pkg/operator/events".Recorder, *"k8s.io/api/rbac/v1".Role)
pkg/operator/controller/configurable-route/controller.go:375:45: not enough arguments in call to resourceapply.ApplyRoleBinding
	have ("k8s.io/client-go/kubernetes/typed/rbac/v1".RbacV1Interface, "github.com/openshift/library-go/pkg/operator/events".Recorder, *"k8s.io/api/rbac/v1".RoleBinding)
	want (context.Context, "k8s.io/client-go/kubernetes/typed/rbac/v1".RoleBindingsGetter, "github.com/openshift/library-go/pkg/operator/events".Recorder, *"k8s.io/api/rbac/v1".RoleBinding)
```